etc/profile-m-z/telegram.profile

```diff
@@ -16,9 +15,7 @@ include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
 
-mkdir ${HOME}/.TelegramDesktop
 mkdir ${HOME}/.local/share/TelegramDesktop
-whitelist ${HOME}/.TelegramDesktop
```

Comment:
If you did not started telegram for a long time, this directory might still be read if you start telegram to migrate the settings.